src/System.Security.Cryptography.Cng/tests/System.Security.Cryptography.Cng.Tests.csproj

```diff
@@ -159,6 +148,23 @@
       <Link>CommonTest\System\Security\Cryptography\AlgorithmImplementations\RSA\RSAXml.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'net462' OR '$(TargetGroup)' == 'net47' OR '$(TargetGroup)' == 'netcoreapp'">
```

Comment:
This feels a little weird... like "!= netfx" is the more correct answer.  But hopefully we'll scrub all the places we see "462" once we move the baseline, so this will at least stand out.

Good with me unless Wes has any objection.